### PR TITLE
Change the behavior of `nnictl log trial  [--trialid TRIALID] [id]`

### PR DESCRIPTION
**Pull request by leelaylay**
Related to #886

### CODE REVIEW

File: tools/nni_cmd/nnictl_utils.py [hunk 1]
```diff
@@ -328,13 +328,18 @@ def log_trial(args):
...
+                print_error('trial id is not valid!')
+                exit(1)
         else:
-            print_error('trial id is not valid!')
+            print_error('please specific the trial id!')
+            print_error("trial id list in this experiment: " + str(list(trial_id_path_dict.keys())))
             exit(1)
     else:
-        for key in trial_id_path_dict.keys():
+        for key in trial_id_path_dict:
```
Comment (leelaylay) [inline tools/nni_cmd/nnictl_utils.py]: pylint recommand me to do this

File: tools/nni_cmd/nnictl_utils.py [hunk 2]
```diff
@@ -328,13 +328,18 @@ def log_trial(args):
...
         exit(1)
     if args.id:
-        if trial_id_path_dict.get(args.id):
-            print('id:' + args.id + ' path:' + trial_id_path_dict[args.id])
+        if args.trialid:
+            if trial_id_path_dict.get(args.trialid):
+                print('id:' + args.trialid + ' path:' + trial_id_path_dict[args.trialid])
+            else:
+                print_error('trial id is not valid!')
+                exit(1)
```
Comment (leelaylay) [inline tools/nni_cmd/nnictl_utils.py]: original `args.trialid` doesn't work at all

File: tools/nni_cmd/nnictl_utils.py [hunk 3]
```diff
@@ -328,13 +328,18 @@ def log_trial(args):
         print_error('Restful server is not running...')
         exit(1)
     if args.id:
```
Comment (SparkSnail) [inline tools/nni_cmd/nnictl_utils.py]: what's the effect of args.id here?

Comment (leelaylay) [reply tools/nni_cmd/nnictl_utils.py]: @SparkSnail 
To check whether experiment id exists. If not, print all possible trial id list & log_path.

File: docs/zh_CN/NNICTLDOC.md [hunk 4]
```diff
@@ -360,7 +360,8 @@ nnictl 支持的命令：
   
   | 参数及缩写 | 是否必需  | 默认值 | 说明         |
   | ----- | ----- | --- | ---------- |
-  | id    | False |     | Trial 的 id |
+  | id         | False |     | 需要设置的 Experiment 的 id. 当有多个 Experiments 时，需要填写该参数 |
```
Comment (chicm-ms) [inline docs/zh_CN/NNICTLDOC.md]: @squirrelsc  I assume we do not need to update Chinese version?

Comment (squirrelsc) [reply docs/zh_CN/NNICTLDOC.md]: Update it in crowd in later, after this submitted. Don't change Chinese directly.

Comment (leelaylay) [reply docs/zh_CN/NNICTLDOC.md]: Sorry. I discard this change to Chinese doc.

File: docs/en_US/NNICTLDOC.md [hunk 5]
```diff
@@ -354,7 +354,8 @@ nnictl support commands:
 
   |Name, shorthand|Required|Default|Description|
   |------|------|------ |------|
-  |id| False| |the id of trial|
+  |id|  False| |ID of the experiment you want to set. Required if there are multiple experiments running.|
```
[the remark this answers is not in the document]
Comment (leelaylay) [reply docs/en_US/NNICTLDOC.md]: ID means trial id and replace trialid as experimentID now.

**Pull request merged**